.github/workflows/turing.yaml

```diff
@@ -374,10 +374,10 @@ jobs:
         with:
           name: cluster-init.${{ env.CLUSTER_INIT_VERSION }}.tar
 
-      - name: Set up Go 1.14
+      - name: Set up Go 1.16
         uses: actions/setup-go@v2
         with:
-          go-version: "1.14"
+          go-version: "1.16"
```

Comment:
There are two `Set up Go` steps in this job (probably it's an artefact of the merge op). This one can be removed